lib/p2p/Pool.ts

```diff
@@ -80,6 +82,14 @@ class Pool extends EventEmitter {
     this.connected = false;
   }
 
+  private seedPeers = async (): Promise<void> => {
+    await Promise.all([
+      this.hosts.getOrCreateHost(Peer.fromOutbound(new SocketAddress('xud1.test.exchangeunion.com', 8885))),
+      this.hosts.getOrCreateHost(Peer.fromOutbound(new SocketAddress('xud2.test.exchangeunion.com', 8885))),
+      this.hosts.getOrCreateHost(Peer.fromOutbound(new SocketAddress('xud3.test.exchangeunion.com', 8885))),
```

Comment:
I don't think this will work actually, because calling `fromOutbound` connects to the peer and if we're already connected to the seed peers we'd be attempting to connect to them twice here. Maybe the best option is putting this logic in `DB.createDatabase()` because there we're sure we're dealing with a fresh database and won't be attempting to duplicate the seed peers.